java/src/main/java/io/vertx/ext/eventbus/client/transport/Transport.java

```diff
@@ -0,0 +1,227 @@
+package io.vertx.ext.eventbus.client.transport;
+
+import io.netty.channel.*;
+import io.netty.handler.proxy.*;
+import io.netty.handler.ssl.SslHandler;
+import io.netty.handler.ssl.util.InsecureTrustManagerFactory;
+import io.netty.handler.timeout.IdleStateEvent;
+import io.netty.handler.timeout.IdleStateHandler;
+import io.netty.util.concurrent.Future;
+import io.netty.util.concurrent.GenericFutureListener;
+import io.vertx.ext.eventbus.client.EventBusClient;
+import io.vertx.ext.eventbus.client.EventBusClientOptions;
+import io.vertx.ext.eventbus.client.Handler;
+import io.vertx.ext.eventbus.client.logging.Logger;
+import io.vertx.ext.eventbus.client.logging.LoggerFactory;
+import io.vertx.ext.eventbus.client.options.ProxyOptions;
+import io.vertx.ext.eventbus.client.options.ProxyType;
+import io.vertx.ext.eventbus.client.options.TrustOptions;
+
+import javax.net.ssl.*;
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
+import java.security.KeyStore;
+import java.security.SecureRandom;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author <a href="mailto:julien@julienviet.com">Julien Viet</a>
+ */
+public abstract class Transport extends ChannelInitializer {
+
+  protected EventBusClientOptions options;
```

Comment:
relevant options field should be inline in the class instead and made final.

Reply:
I made it final, but could you elaborate what you mean with the first part?